src/RequestLogger/RequestLogger.cs

```diff
@@ -157,7 +154,8 @@ private static void LogRequest(HttpContext context, CorrelationVector cv, double
                     { "Verb", context.Request.Method },
                     { "Path", GetPathAndQuerystring(context.Request) },
                     { "Host", context.Request.Headers["Host"].ToString() },
-                    { "ClientIP", GetClientIp(context) },
+                    { "ClientIP", GetClientIp(context, out string xff) },
+                    { "XFF", xff },
                     { "UserAgent", context.Request.Headers["User-Agent"].ToString() },
```

Comment:
added XFF header to the log - this lets us see all the proxies the request went through